frontend/src/pages/CodeEditor.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState } from 'react';
+import Editor from '@monaco-editor/react';
+import { Grid } from '@mui/material';
+import { exampleQuestion1 } from './Dashboard';
+import { CodeEvaluator } from '../utils/codeEvaluator';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import * as ts from 'typescript';
```

Comment:
We probably need to specify typescript as a dependency in package.json

Reply:
i think it already is declared

Reply:
Its only declared as a dev dependency. If we build it, i think this will fail, since typescript wont be packaged

Reply:
added